<!DOCTYPE html>
<html lang="en">
    
    <head>
    <link href="https://gmpg.org/xfn/11" rel="profile">
    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="Cache-Control" content="public" />
<!-- Enable responsiveness on mobile devices -->
<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">
<meta name="generator" content="Hugo 0.74.3" />

    
    
    

<title>Projects • Ahmed Hasan</title>


<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Projects"/>
<meta name="twitter:description" content=""/>

<meta property="og:title" content="Projects" />
<meta property="og:description" content="" />
<meta property="og:type" content="website" />
<meta property="og:url" content="https://aays.github.io/projects/" />
<meta property="og:updated_time" content="2020-03-17T00:00:00+00:00" /><meta property="og:site_name" content="Title" />


    


<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">








<link rel="stylesheet" href="/scss/hyde-hyde.4e02f3d31db978c29e43cbdee3af32ed98d01bfbdf98034eee6f1408857c85bf.css" integrity="sha256-TgLz0x25eMKeQ8ve468y7ZjQG/vfmANO7m8UCIV8hb8=">


<link rel="stylesheet" href="/scss/print.2744dcbf8a0b2e74f8a50e4b34e5f441be7cf93cc7de27029121c6a09f9e77bc.css" integrity="sha256-J0Tcv4oLLnT4pQ5LNOX0Qb58&#43;TzH3icCkSHGoJ&#43;ed7w=" media="print">




<link rel="stylesheet" href="/scss/tocbot.5ef07cebc3c477b54270456f149ee02922479bb7555fd344b2c69f953b0e7e5e.css" integrity="sha256-XvB868PEd7VCcEVvFJ7gKSJHm7dVX9NEssaflTsOfl4=">



    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
    <!-- Icons -->
    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/apple-touch-icon-144-precomposed.png">
    <link rel="shortcut icon" href="/favicon.png">
    <link rel="alternate" type="application/rss+xml" href="https://aays.github.io/projects/index.xml" title="Ahmed Hasan" />

    

</head>


    <body class=" ">
    
<div class="sidebar">
  <div class="container ">
    <div class="sidebar-about">
      <span class="site__title">
        <a href="https://aays.github.io/">Ahmed Hasan</a>
      </span>
      
        
        
        
        <div class="author-image">
          <img src="https://aays.github.io/img/profile.jpeg" alt="Author Image" class="img--circle img--headshot element--center">
        </div>
        
      
      
      <p class="site__description">
        
      </p>
    </div>
    <div class="collapsible-menu">
      <input type="checkbox" id="menuToggle">
      <label for="menuToggle">Ahmed Hasan</label>
      <div class="menu-content">
        <div>
	<ul class="sidebar-nav">
		 
		 
			 
				<li>
					<a href="/about/">
						<span>About</span>
					</a>
				</li>
			 
		 
			 
				<li>
					<a href="/post/">
						<span>Posts</span>
					</a>
				</li>
			 
		 
			 
				<li>
					<a href="/projects/">
						<span>Projects</span>
					</a>
				</li>
			 
		 
			 
				<li>
					<a href="/vitae/">
						<span>CV</span>
					</a>
				</li>
			 
		
	</ul>
</div>

        <section class="social">
	
	<a href="https://twitter.com/ahmedrhasan" rel="me"><i class="fab fa-twitter fa-lg" aria-hidden="true"></i></a>
	
	
	
	<a href="https://github.com/aays" rel="me"><i class="fab fa-github fa-lg" aria-hidden="true"></i></a>
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	<a href="mailto:ahmed.hasan@mail.utoronto.com" rel="me"><i class="fas fa-at fa-lg" aria-hidden="true"></i></a>
	
</section>

      </div>
    </div>
    
<div class="copyright">
  &copy; 2021 htr3n
  
    <a href="https://creativecommons.org/licenses/by-sa/4.0">CC BY-SA 4.0</a>
  
</div>


<div class="builtwith">
Built with <a href="https://gohugo.io">Hugo</a> and <a href="https://github.com/htr3n/hyde-hyde">hyde-hyde</a>.
</div>


  </div>
</div>

        <div class="content container">
            
    <span class="section__title">Projects</span>
<div class="portfolio__content">
    <h2>Graduate Research</h2>
    <section>
        <div class="portfolio_content">
            <hr class="divider">
            
            <div class="row">
                <div class="col-md-4 col-sm-4 col-xs-12" href="https://academic.oup.com/gbe/article/12/4/370/5809146"
                    target="_blank" rel="noopener noreferrer">
                    <img class="project__image img-responsive" src="ld-paper.png" alt="Recombination rate variation and infrequent sex influence genetic diversity in C. reinhardtii">
                </div>
                <div class="col-md-8 col-sm-8 col-xs-12">
                    <span class="project__title">
                        <a href="https://academic.oup.com/gbe/article/12/4/370/5809146" target="_blank" rel="noopener noreferrer">Recombination rate variation and infrequent sex influence genetic diversity in C. reinhardtii</a>
                    </span>
                    
                    <span class="project__summary">
                        <p>Recombination confers a major evolutionary advantage by breaking up linkage
disequilibrium between harmful and beneficial mutations, thereby facilitating
selection. However, in species that are only periodically sexual, such as many
microbial eukaryotes, the realized rate of recombination is also affected by the
frequency of sex, meaning that infrequent sex can increase the effects of
selection at linked sites despite high recombination rates. Despite this, the
rate of sex of most facultatively sexual species is unknown. Here, we use
genomewide patterns of linkage disequilibrium to infer fine-scale recombination
rate variation in the genome of the facultatively sexual green alga
<em>Chlamydomonas reinhardtii</em>. We observe recombination rate variation of up to two
orders of magnitude and find evidence of recombination hotspots across the
genome. Recombination rate is highest flanking genes, consistent with trends
observed in other nonmammalian organisms, though intergenic recombination rates
vary by intergenic tract length. We also find a positive relationship between
nucleotide diversity and physical recombination rate, suggesting a widespread
influence of selection at linked sites in the genome. Finally, we use estimates
of the effective rate of recombination to calculate the rate of sex that occurs
in natural populations, estimating a sexual cycle roughly every 840 generations.
We argue that the relatively infrequent rate of sex and large effective
population size creates a population genetic environment that increases the
influence of selection on linked sites across the genome.</p>

                    </span>
                </div>
            </div>
            <div class="row-space">&nbsp;</div>
            
            <div class="row">
                <div class="col-md-4 col-sm-4 col-xs-12" href="http://aays.github.io/post/mt-locus/"
                    target="_blank" rel="noopener noreferrer">
                    <img class="project__image img-responsive" src="mt-locus.png" alt="Recombination in the C. reinhardtii MT locus and consequences for selection efficacy">
                </div>
                <div class="col-md-8 col-sm-8 col-xs-12">
                    <span class="project__title">
                        <a href="http://aays.github.io/post/mt-locus/" target="_blank" rel="noopener noreferrer">Recombination in the C. reinhardtii MT locus and consequences for selection efficacy</a>
                    </span>
                    
                    <span class="project__summary">
                        <p>‘Sex’ in <em>C. reinhardtii</em> is determined by a dimorphic, crossover-suppressed
mating type locus, which encodes an individual as either <em>MT+</em> or <em>MT-</em>. <em>C.
reinhardtii</em> is isogamous (equal sized gametes) which means there are no sex
differences; hence the term ‘mating type’. The two alleles of the <em>MT</em> locus can
be thought of as ‘X and Y chromosomes’, except that each individual only
carries one and that the alleles are genetically very similar.</p>
<p>Unexpectedly similar, that is. Work has shown that the two <em>MT</em> alleles of <em>C.
reinhardtii</em> are hardly differentiated from one another. Meanwhile, its
multicellular, anisogamous relative <em>Volvox carteri</em> has shown over 100x more
differentiation between its own <em>MT</em> alleles (Ferris 2010). Later work showed
evidence for gene conversion, a form of noncrossover recombination, in two <em>C.
reinhardtii</em> genes that had copies in both <em>MT</em> alleles, suggesting that genetic
exchange was preventing the two alleles from diverging.</p>
<p>For this project, we estimate levels of genetic exchange across the <em>MT</em> locus of
<em>C. reinhardtii</em>, and undertake a population genetic analysis to find out whether
gene conversion is sufficient to avoid the differentiation and degeneration
that is often seen in sex determining loci, asking the questions:</p>
<ol>
<li>What are the spatial patterns of LD across <em>MT</em>?</li>
<li>Do patterns of polymorphism or divergence between <em>MT</em> alleles reflect the
strength of LD?</li>
<li>Do shared and <em>MT</em>-limited regions show evidence for differences in selection
efficacy?</li>
<li>Do patterns of inter-chromosomal LD between mating type alleles and organelle
genomes suggest leaky uniparental inheritance?</li>
</ol>

                    </span>
                </div>
            </div>
            <div class="row-space">&nbsp;</div>
            
        </div>
    </section>
</div>
<div class="portfolio__content">
    <h2>Pedagogical Work</h2>
    <section>
        <div class="portfolio_content">
            <hr class="divider">
            
            <div class="row">
                <div class="col-md-4 col-sm-4 col-xs-12" href="https://jose.theoj.org/papers/10.21105/jose.00049.pdf"
                    target="_blank" rel="noopener noreferrer">
                    <img class="project__image img-responsive" src="rcourse.png" alt="Quantitative Methods in R for Biology - a live-coding undergraduate course">
                </div>
                <div class="col-md-8 col-sm-8 col-xs-12">
                    <span class="project__title">
                        <a href="https://jose.theoj.org/papers/10.21105/jose.00049.pdf" target="_blank" rel="noopener noreferrer">Quantitative Methods in R for Biology - a live-coding undergraduate course</a>
                    </span>
                    
                    <span class="project__summary">
                        <p>During the Fall 2018 and Fall 2019 semesters, I was part of the instructor team
for a third-year course on quantitative methods in R, pitched by <a href="https://uoftcoders.github.io">UofT
Coders</a> and adapted from Dr. Christie Bahlai&rsquo;s
<a href="https://cbahlai.github.io/rqm-template/">Reproducible Quantitative Methods</a>
course. The course aims to teach undergraduate students at the third-year level
or above the foundations for reproducible data analysis in R, and is taught
entirely using the live-coding format used in <a href="https://carpentries.org">Carpentries</a>
workshops. The course material is openly available on the <a href="https://uoftcoders.github.io/rcourse">course
website</a>.</p>

                    </span>
                </div>
            </div>
            <div class="row-space">&nbsp;</div>
            
        </div>
    </section>
</div>
<div class="portfolio__content">
    <h2>Undergraduate Research</h2>
    <section>
        <div class="portfolio_content">
            <hr class="divider">
            
            <div class="row">
                <div class="col-md-4 col-sm-4 col-xs-12" href=""
                    target="_blank" rel="noopener noreferrer">
                    <img class="project__image img-responsive" src="cily.png" alt="Endophytes as biocontrols for the Côte d’Ivoire Lethal Yellowing (CILY) disease">
                </div>
                <div class="col-md-8 col-sm-8 col-xs-12">
                    <span class="project__title">
                        <a href="" target="_blank" rel="noopener noreferrer">Endophytes as biocontrols for the Côte d’Ivoire Lethal Yellowing (CILY) disease</a>
                    </span>
                    
                    <span class="project__summary">
                        <p>As part of my undergraduate research work, I attempted to identify endophytic
microbes that may potentially be used as biocontrols against the Côte d’Ivoire
Lethal Yellowing (CILY) disease, which had previously destroyed approximately
350 ha of coconut trees in the West African country. This was done by creating
profiles of the microbial communities present within both infected and
uninfected trees. A ‘dilution-to-extinction’ method of high-throughput
culturing was used in order to cultivate both bacterial and fungal endophytes
directly from samples collected from the town of Grand-Lahou, followed by
molecular analyses for strain identification. Our results identified
<em>Trichoderma</em>, <em>Penicillium</em>, <em>Bacillus</em>, and <em>Pseudomonas</em> as endophytes
present within coconut palms. Several of these endophytes have historically
been used as part of microbial biocontrol consortia against other plant
pathogens such as <em>Fusarium</em>, and show promise as potential biocontrols to be
tested for use against CILY in field studies.</p>
<p>Related publication: Morales-Lizcano et al. (2017) Microbial diversity in
leaves, trunk and rhizosphere of coconut palms (Cocos nucifera L.) associated
with the coconut lethal yellowing phytoplasma in Grand-Lahou, Côte d’Ivoire.
<a href="http://www.academicjournals.org/journal/AJB/article-abstract/BE373F465038">Available online</a>.</p>
<p>Bonus: Here’s a fun and largely jargon-free <a href="https://www.youtube.com/watch?v=59eKe4kp1bU">one-minute video
summary</a> of the project that I
made.</p>

                    </span>
                </div>
            </div>
            <div class="row-space">&nbsp;</div>
            
        </div>
    </section>
</div>


        </div>
        
    
<script defer src="https://use.fontawesome.com/releases/v5.5.0/js/all.js" integrity="sha384-GqVMZRt5Gn7tB9D9q7ONtcp4gtHIUEW/yG7h98J7IpE3kpi+srfFyyB/04OV6pG0" crossorigin="anonymous"></script>



    </body>
</html>
