<!DOCTYPE html>
<html lang="en">
    
    


    <head>
    <link href="https://gmpg.org/xfn/11" rel="profile">
    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="Cache-Control" content="public" />
<!-- Enable responsiveness on mobile devices -->
<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">
<meta name="generator" content="Hugo 0.60.0" />

    
    
    

<title>Consequences of recombination for mating type evolution in C. reinhardtii - a summary • Ahmed Hasan</title>


<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Consequences of recombination for mating type evolution in C. reinhardtii - a summary"/>
<meta name="twitter:description" content="Motivation Volvocine algae have been a model lineage for studying the evolutionary transition from isogamy (equal sized gametes) to anisogamy (unequal sized gametes). Instead of entire sex chromosomes, isogamous systems tend to have smaller and less differentiated mating type (MT) loci. Part of this putatively stems from these systems simply having less secondary sexual characteristics, and thus comparatively minimal need to segregate certain genes in one MT allele over another (under the sexually antagonistic selection model)."/>

<meta property="og:title" content="Consequences of recombination for mating type evolution in C. reinhardtii - a summary" />
<meta property="og:description" content="Motivation Volvocine algae have been a model lineage for studying the evolutionary transition from isogamy (equal sized gametes) to anisogamy (unequal sized gametes). Instead of entire sex chromosomes, isogamous systems tend to have smaller and less differentiated mating type (MT) loci. Part of this putatively stems from these systems simply having less secondary sexual characteristics, and thus comparatively minimal need to segregate certain genes in one MT allele over another (under the sexually antagonistic selection model)." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://aays.github.io/post/mt-locus/" />
<meta property="article:published_time" content="2019-03-05T00:00:00+00:00" />
<meta property="article:modified_time" content="2019-12-18T22:31:55-05:00" /><meta property="og:site_name" content="Title" />


    


<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">








<link rel="stylesheet" href="/scss/hyde-hyde.1b33b942c41726c5f2c4b4b6b87d1139357c3f0d30f7643a3123e6bce1aa5255.css" integrity="sha256-GzO5QsQXJsXyxLS2uH0ROTV8Pw0w92Q6MSPmvOGqUlU=">


<link rel="stylesheet" href="/scss/print.2744dcbf8a0b2e74f8a50e4b34e5f441be7cf93cc7de27029121c6a09f9e77bc.css" integrity="sha256-J0Tcv4oLLnT4pQ5LNOX0Qb58&#43;TzH3icCkSHGoJ&#43;ed7w=" media="print">




<link rel="stylesheet" href="/scss/tocbot.126e7d75240acc946d34d6ccb2982ed4c394a5fe34e73e1a2119ca951bcf119d.css" integrity="sha256-Em59dSQKzJRtNNbMspgu1MOUpf405z4aIRnKlRvPEZ0=">



    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
    <!-- Icons -->
    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/apple-touch-icon-144-precomposed.png">
    <link rel="shortcut icon" href="/favicon.png">
    
    

    
</head>


    <body class=" ">
    
<div class="sidebar">
  <div class="container ">
    <div class="sidebar-about">
      <span class="site__title">
        <a href="https://aays.github.io/">Ahmed Hasan</a>
      </span>
      
        
        
        
        <div class="author-image">
          <img src="https://aays.github.io/img/profile.jpeg" alt="Author Image" class="img--circle img--headshot element--center">
        </div>
        
      
      
      <p class="site__description">
        
      </p>
    </div>
    <div class="collapsible-menu">
      <input type="checkbox" id="menuToggle">
      <label for="menuToggle">Ahmed Hasan</label>
      <div class="menu-content">
        <div>
	<ul class="sidebar-nav">
		 
		 
			 
				<li>
					<a href="/about/">
						<span>About</span>
					</a>
				</li>
			 
		 
			 
				<li>
					<a href="/post/">
						<span>Posts</span>
					</a>
				</li>
			 
		 
			 
				<li>
					<a href="/projects/">
						<span>Projects</span>
					</a>
				</li>
			 
		 
			 
				<li>
					<a href="/vitae/">
						<span>CV</span>
					</a>
				</li>
			 
		
	</ul>
</div>

        <section class="social">
	
	<a href="https://twitter.com/ahmedrhasan" rel="me"><i class="fab fa-twitter fa-lg" aria-hidden="true"></i></a>
	
	
	
	<a href="https://github.com/aays" rel="me"><i class="fab fa-github fa-lg" aria-hidden="true"></i></a>
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	<a href="mailto:ahmed.hasan@mail.utoronto.com" rel="me"><i class="fas fa-at fa-lg" aria-hidden="true"></i></a>
	
</section>

      </div>
    </div>
    
<div class="copyright">
  &copy; 2019 htr3n
  
    <a href="https://creativecommons.org/licenses/by-sa/4.0">CC BY-SA 4.0</a>
  
</div>


<div class="builtwith">
Built with <a href="https://gohugo.io">Hugo</a> and <a href="https://github.com/htr3n/hyde-hyde">hyde-hyde</a>.
</div>


  </div>
</div>

        <div class="content container">
            
    
<article>
  <header>
    <h1>Consequences of recombination for mating type evolution in C. reinhardtii - a summary</h1>
    
    
<div class="post__meta">
    
    
      <i class="fas fa-calendar-alt"></i> Mar 5, 2019
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/publications">PUBLICATIONS</a>
              
          
      
    
    
    
      
      
          <br/>
           <i class="fas fa-tags"></i>
          
          <a class="badge badge-tag" href="/tags/research">research</a>
           
      
          <a class="badge badge-tag" href="/tags/writing">writing</a>
          
      
    
    
    <br/>
    
</div>


  </header>
  
  
  
    <div class="toc-wrapper">
      <input type="checkbox" id="tocToggle">
      <label for="tocToggle">Table of Contents</label>
      
        <div class="toc" id="TableOfContents"></div>
      
    </div>
  
  
  <div class="post">
    


<div id="motivation" class="section level2">
<h2>Motivation</h2>
<p>Volvocine algae have been a model lineage for studying the evolutionary
transition from isogamy (equal sized gametes) to anisogamy (unequal sized
gametes). Instead of entire sex chromosomes, isogamous systems tend to have
smaller and less differentiated mating type (MT) loci. Part of this putatively
stems from these systems simply having less secondary sexual characteristics,
and thus comparatively minimal need to segregate certain genes in one MT allele
over another (under the <em>sexually antagonistic selection</em> model).</p>
<p>In the unicellular isogamous alga <em>Chlamydomonas reinhardtii</em>, mating types
(i.e. sexes) are determined by a dimorphic mating type locus. The two MT
alleles (these are analogous to ‘X and Y chromosomes’) are termed MT+ (~650 kb)
and MT- (~450 kb) and differ in size, gene content, and gene order. That being
said, the vast majority of their genes are shared across both alleles. Unlike
in some yeast species, the MT locus of <em>C. reinhardtii</em> does not switch;
i.e. an individual’s mating type is genetically encoded and immutable. The MT locus
is subdivided into three domains; centremost is the rearranged (R) domain,
containing numerous inversions and structural variants. Flanking the R domain
are the more syntenic telomere-proximal (T) and centromere-proximal (C)
domains. Here’s a great schematic of the MT locus from <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1003724">De Hoff et al., 2013</a>:</p>
<div class="figure">
<img src="https://github.com/aays/aays.github.io/blob/master/post/mt-locus/mt_locus.png?raw=true" alt="MT image" />
<p class="caption">MT image</p>
</div>
<p>Previous research showed that these shared genes are two whole orders of
magnitude (!) less diverged than the MT locus of <em>C. reinhardtii</em>’s anisogamous
multicellular relative <em>Volvox carteri</em> (Ferris 2010, Science). This means that
the ‘X and Y’ regions of <em>V. carteri</em> were about 100x more different from one
another than those of <em>C. reinhardtii</em>! This suggested that while MT alleles in
<em>V. carteri</em> diverged from one another over time, as is expected of most sex
chromosome systems, the MT alleles of <em>C. reinhardtii</em> maintained similarity
between the two MT alleles despite being as evolutionarily old as <em>V. carteri</em>
if not older.</p>
<p>So what was causing this surprising lack of differentiation? Work by De Hoff et
al. (2013) showed evidence for gene conversion, a form of recombination that
happens over short distances and does not require crossovers, in two of the
shared genes in <em>C. reinhardtii</em>. This suggested that it was in fact gene
conversion that was reducing differentiation between MT alleles by allowing
sequence from MT allele to be shared with the other.</p>
</div>
<div id="our-study" class="section level2">
<h2>Our study</h2>
<p>Although gene conversion could potentially explain the lack of differentiation
between MT alleles, it had yet to be tested whether gene conversion is
sufficient to avoid the degeneration of functional sequence.</p>
<p>To assess levels of gene conversion, we calculated linkage disequilibrium (LD)
across shared regions in the MT locus of <em>C. reinhardtii</em> as a proxy for
recombination, such that regions of high LD were taken to have low
recombination and vice versa. Linkage disequilibrium can be thought of as the covariance of
alleles; i.e. if two SNP alleles are seen together more often than we’d expect by chance.
Recombination can break down these associations between alleles, thus reducing LD.</p>
<p>Our dataset consisted of 19 whole genome
sequences (9 MT+, 10 MT-) sampled in Quebec. The similarity between the MT
alleles led to some difficulties in read alignment that are detailed in the
manuscript - but it was nothing that a bit of masking couldn’t fix!</p>
<p>Next, we asked the questions:</p>
<ol style="list-style-type: decimal">
<li><p>What are the spatial patterns of LD across MT and flanking autosomal sequence?</p></li>
<li><p>Do patterns of polymorphism or divergence between MT alleles reflect the strength of LD?</p></li>
<li><p>Do shared and MT-limited regions show evidence for differences in selection efficacy?</p></li>
</ol>
<p>An ancillary analysis we performed involved estimating inter-chromosomal LD between
MT alleles and organelle genomes. This is because organelle inheritance is believed
to be uniparental in <em>C. reinhardtii</em>, as in many other plants, but evidence for
leakage has been seen in plastid inheritance (Sager 1954, Ness 2016). I won’t be
covering the organelle inheritance angle much in this blog post as I’m already
concerned about the length of this one, but it would be a great topic for a
separate post in the future!</p>
<p><a href="https://www.biorxiv.org/content/10.1101/565275v1">Here’s a link to the paper</a>.</p>
</div>
<div id="alignment-method" class="section level2">
<h2>Alignment method</h2>
<p>I won’t be going too in depth about the methods as well here, as they’ve been
detailed in the paper, but I did want to briefly talk about the alignment method
we used here to assess LD across MT.</p>
<p>With the two MT alleles having very different sizes, simple alignment of the entire
alleles as is wasn’t quite feasible; furthermore, there are inversions and
translocations in the MT alleles as well that needed to be taken into account.</p>
<p>With these in mind, I first performed an alignment with the LASTZ whole
genome aligner (Harris 2007, thesis) with the MT+ allele as the reference
and the MT- as the query. We opted to use the MT+ allele (and its genomic
positions) as the ‘reference’ throughout this study, as it is the longer of the two.</p>
<p>Once LASTZ returned the locations of alignments, my next goal was to
essentially create a series of ‘chunks’ in FASTA format containing sequences
for an MT+ region that had a match in the alignment as well as the matching MT-
sequences. Following some bioinformatic wrangling, this resulted in a large MT+
sized ‘master alignment’, but with all non-shared regions masked. All shared
regions, meanwhile, contained both MT+ and aligned MT- sequence from the LASTZ
alignment, enabling the calculation of LD, F<sub>ST</sub>, and more across
shared regions. Scripts for this alignment process can be found in the <a href="https://github.com/aays/2019-mt-locus/tree/master/analysis/alignment-lastz">project
repo</a>.</p>
</div>
<div id="results" class="section level2">
<h2>Results</h2>
<div id="breakdown-of-ld-across-the-c-and-t-domains" class="section level3">
<h3>Breakdown of LD across the C and T domains</h3>
<p>First, we looked at LD in and out of the three domains of the MT locus as well
as the surrounding autosomal regions. The measure we used is called ZnS (Kelly
1997), which is the average of all pairwise estimates of LD in a region. 1
indicates complete LD (i.e. no recombination) while 0 indicates no LD
whatsoever (i.e. high recombination).</p>
<p>The pattern is quite stark - LD is very high in the R domain, which is expected
given that that region contains many structural variants, which suppress
recombination. The two more syntenic domains, however, show LD levels that
approximate autosomal levels. Interestingly, we don’t quite see a clear
boundary for recombination suppression in the R domain, even after applying a
three state hidden Markov Model. It’s also worth noting that LD in the R domain
is still not fixed at 1 across windows, despite the suppression of crossovers.</p>
</div>
<div id="differentiation-between-shared-regions-inversely-scaled-with-recombination-rate" class="section level3">
<h3>Differentiation between shared regions inversely scaled with recombination rate</h3>
<p>Next, we calculated differentiation (F<sub>ST</sub>) across 1 kb windows of
sequence and compared its levels of LD. A strong correlation here is expected;
by definition, as differentiation increases, so too does the level at which
alleles covary within each of the two ‘populations’ (MT alleles, in this case)
under consideration, and this is exactly what we see.</p>
</div>
<div id="genes-with-higher-recombination-rates-exhibit-higher-selection-efficacy-and-gc-content" class="section level3">
<h3>Genes with higher recombination rates exhibit higher selection efficacy and GC content</h3>
<p>The central question of our study, however, had to do with whether gene
conversion could prevent degeneration in the MT locus. Before we get into the
result, let’s talk about sex chromosome degeneration for a bit.</p>
<p>‘Degeneration’ itself specifically refers to gene loss from sex chromosomes,
usually in the context of Y chromosome evolution (Bachtrog 2013) but has been
more informally used to describe the negative consequences of recombination
suppression. Less recombination means inefficient selection due to what are
known as Hill-Robertson effects, where selection at one site interferes with
selection at one or more linked sites (see Comeron 2008 for review). Thus,
recombination-suppressed sex chromosomes also tend to accumulate repeat
regions, chromosomal inversions, autosomal translocations, and transposable
elements, usually due to weaker selection.</p>
<p>Muller’s ratchet effects are also theorized to cause degeneration. This is the
idea that in the absence of recombination, the only force generating variation
is mutation, and mutations are largely deleterious. With this in mind, the
‘least mutated haplotype’ in a population at any given point in time may be
lost in the next generation due to genetic drift, which results in the ‘next
least mutated haplotype’ now having the honour of being the least mutated one
for the time being. As time wears on, mutations irreversibly accumulate.</p>
<p>Gene conversion, however, bears the potential of counteracting both of these
processes. Recombination can uncouple linked sites and allow for more efficient
selection. To investigate that here, we calculated selection efficacy as
π<sub>N</sub> over π<sub>S</sub>; i.e. nucleotide diversity at nonsynonymous
sites over nucleotide diversity at synonymous sites. This ratio is expected to
be below 1 under purifying selection, which is expected given that most
mutations are deleterious. However, <em>how much smaller than 1</em> it is reflects
the strength of purifying selection. Very low values suggest nonsynonymous
mutations are being purged efficiently while synonymous diversity is preserved.</p>
<p>We calculated π<sub>N</sub> / π<sub>S</sub> for all genes across the MT locus,
and compared them to LD levels for shared genes. We observe that as LD goes up
(i.e. recombination rates go down) π<sub>N</sub> / π<sub>S</sub> values
increase, suggesting weaker selection efficacy. This suggests that genes with
greater rates of genetic exchange show more efficient selection! (Fig. 3b in
the paper) We also find no significant difference between selection efficacy in
shared MT genes and 100 randomly sampled autosomal genes, which implies that
gene conversion is sufficient to preserve an analogous level of selection
efficacy!</p>
<p>Another angle from which to look at recombination and selection at linked sites
is to look at the correlation between diversity at selectively unconstrained
sites and recombination rates. Since Hill-Robertson effects result in the
reduction of putatively neutral diversity in regions of low recombination, we expect
diversity to scale with recombination (Begun and Aquadro 1992) which we see
in both mating type alleles (Fig. 3a).</p>
</div>
</div>
<div id="so-what-does-this-mean" class="section level2">
<h2>So what does this mean?</h2>
<p>These results point to gene conversion both reducing differentiation
and improving selection efficacy in shared genes. So where do these
results fit in the context of what we know about MT evolution
in Volvocine algae?</p>
<p>Let’s consider <em>V. carteri</em> once again; it is anisogamous and features high
levels of differentiation between its two MT alleles as compared to the MT
locus of <em>C. reinhardtii</em>. Differentiation between the two <em>V. carteri</em> MT
alleles likely occurred after it had already transitioned to anisogamy (Hiraide
2013, Hamaji 2018). This suggests that selection for recombination suppression
followed selective pressure on sex-specific traits, implying that reduced gene
conversion was a factor in the differentiation of the <em>V. carteri</em> MT locus.</p>
<p>Low differentiation between mating type alleles has been seen in a number of
isogamous algae besides <em>C. reinhardtii</em> (Hiraide 2013, Hamaji 2016). In
comparing the MT locus of <em>C. reinhardtii</em> to <em>V. carteri</em>, we argue these
findings mean that gene conversion is a key mechanism in preventing
differentiation and potential degeneration in isogamous systems.</p>
</div>
<div id="a-technical-note---working-open-and-with-reproducibility-in-mind" class="section level2">
<h2>A technical note - working open and with reproducibility in mind</h2>
<p>For this project, I took it upon myself to work as openly as possible: i.e. all
my scripts (in progress or completed) as well as my ‘lab notebook’-style notes
to myself were made publicly available as they were written, and can still be
found in their entirety in the <a href="https://github.com/aays/2019-mt-locus">project repo</a>. This includes all
my many missteps and scripting bugs along the way, and was a bit nerve wracking
to commit to! But not only did it force me to work in a more organized fashion,
it also meant that all my notes were there for my own later reference and that
I could easily pull earlier versions of scripts via Git commands. My workflow
was generally to have a tmux window with three panes – a Markdown ‘log’ file
in vim, a bash terminal that doubled as a Python/R interpreter
(<a href="https://github.com/prompt-toolkit/ptpython">ptpython</a> is phenomenal for this), and a vim window
where my scripts were written – and this setup basically carried me
through the entire project.</p>
<p>Now that I’m on the other side of this experience, I can’t recommend it enough;
it’s definitely more work, but made the final stretches of the project far more
manageable since I’d essentially forced myself to be as organized and
detail-oriented as possible. Moreover, I tried to minimize the amount of
hardcoding that went into my Python/R scripts so that anyone trying to adapt my
scripts would have less pathing issues and more to work around. Every function
is documented to the best of my ability and each analysis is coupled with a
shell script that reruns it in order. I haven’t been able to upload all of the
data itself to GitHub (I’m a little reluctant to do so with data files at the
order of gigabytes…) but I’m going to look into how I can make data beyond
just the short read files available upon publication.</p>
<p>I do still think there are a few things to be ironed out as far as making this
project wholly reproducible.I wanted to work towards the ideal of creating
a repo that allows you to ‘fully regenerate the paper’ via a single script,
and didn’t quite meet that lofty goal. Still, this has been an excellent
learning experience and one I would strongly suggest anyone in bioinformatics
to try out for themselves.</p>
<p>(Note: many if not most of the pop gen analyses in the paper were performed by
Rob in his preferred environment of Jupyter Notebooks – these are also available
on the repo)</p>
<p>Anyhow – thank you for reading, and if you have any feedback or comments
on the paper or anything else, please reach out via email or on Twitter!</p>
</div>

  </div>
  


  

  
    


</article>


        </div>
        
    

<script defer src="https://use.fontawesome.com/releases/v5.5.0/js/all.js" integrity="sha384-GqVMZRt5Gn7tB9D9q7ONtcp4gtHIUEW/yG7h98J7IpE3kpi+srfFyyB/04OV6pG0" crossorigin="anonymous"></script>





<script src="https://cdnjs.cloudflare.com/ajax/libs/tocbot/4.4.2/tocbot.js"></script>
<script type="text/javascript">
  if (tocbot) {
    tocbot.init({
      
      tocSelector: '.toc',
      
      contentSelector: '.post',
      
      headingSelector: 'h2, h3, h4',
      collapseDepth: 4
    });
  }
</script>




    



    </body>
</html>
