<!DOCTYPE html>
<html lang="en">
    
    


    <head>
    <link href="https://gmpg.org/xfn/11" rel="profile">
    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="Cache-Control" content="public" />
<!-- Enable responsiveness on mobile devices -->
<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">
<meta name="generator" content="Hugo 0.74.3" />

    
    
    

<title>Recombination rate variation and linked selection in C. reinhardtii v2.0 - a summary • Ahmed Hasan</title>


<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="Recombination rate variation and linked selection in C. reinhardtii v2.0 - a summary"/>
<meta name="twitter:description" content="In June of this year, I uploaded my first first-author (there has to be a cleaner way of saying that) manuscript to the preprint server BioRxiv. This was a huge and important milestone for me, and I was incredibly excited to have it out for the world to see virtually as soon as it was done &ndash; such is the power of preprint servers!
Like in most endeavours in science, however, there&rsquo;s always room for improvement, and I&rsquo;ve been very fortunate to have gotten some thorough feedback from both members of my committee as well as other researchers in the community at large."/>

<meta property="og:title" content="Recombination rate variation and linked selection in C. reinhardtii v2.0 - a summary" />
<meta property="og:description" content="In June of this year, I uploaded my first first-author (there has to be a cleaner way of saying that) manuscript to the preprint server BioRxiv. This was a huge and important milestone for me, and I was incredibly excited to have it out for the world to see virtually as soon as it was done &ndash; such is the power of preprint servers!
Like in most endeavours in science, however, there&rsquo;s always room for improvement, and I&rsquo;ve been very fortunate to have gotten some thorough feedback from both members of my committee as well as other researchers in the community at large." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://aays.github.io/post/ld-paper/" />
<meta property="article:published_time" content="2018-11-05T00:00:00+00:00" />
<meta property="article:modified_time" content="2019-12-18T23:21:34-05:00" /><meta property="og:site_name" content="Title" />


    


<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">








<link rel="stylesheet" href="/scss/hyde-hyde.4e02f3d31db978c29e43cbdee3af32ed98d01bfbdf98034eee6f1408857c85bf.css" integrity="sha256-TgLz0x25eMKeQ8ve468y7ZjQG/vfmANO7m8UCIV8hb8=">


<link rel="stylesheet" href="/scss/print.2744dcbf8a0b2e74f8a50e4b34e5f441be7cf93cc7de27029121c6a09f9e77bc.css" integrity="sha256-J0Tcv4oLLnT4pQ5LNOX0Qb58&#43;TzH3icCkSHGoJ&#43;ed7w=" media="print">




<link rel="stylesheet" href="/scss/tocbot.5ef07cebc3c477b54270456f149ee02922479bb7555fd344b2c69f953b0e7e5e.css" integrity="sha256-XvB868PEd7VCcEVvFJ7gKSJHm7dVX9NEssaflTsOfl4=">



    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
    <!-- Icons -->
    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/apple-touch-icon-144-precomposed.png">
    <link rel="shortcut icon" href="/favicon.png">
    
    

</head>


    <body class=" ">
    
<div class="sidebar">
  <div class="container ">
    <div class="sidebar-about">
      <span class="site__title">
        <a href="https://aays.github.io/">Ahmed Hasan</a>
      </span>
      
        
        
        
        <div class="author-image">
          <img src="https://aays.github.io/img/profile.jpeg" alt="Author Image" class="img--circle img--headshot element--center">
        </div>
        
      
      
      <p class="site__description">
        
      </p>
    </div>
    <div class="collapsible-menu">
      <input type="checkbox" id="menuToggle">
      <label for="menuToggle">Ahmed Hasan</label>
      <div class="menu-content">
        <div>
	<ul class="sidebar-nav">
		 
		 
			 
				<li>
					<a href="/about/">
						<span>About</span>
					</a>
				</li>
			 
		 
			 
				<li>
					<a href="/post/">
						<span>Posts</span>
					</a>
				</li>
			 
		 
			 
				<li>
					<a href="/projects/">
						<span>Projects</span>
					</a>
				</li>
			 
		 
			 
				<li>
					<a href="/vitae/">
						<span>CV</span>
					</a>
				</li>
			 
		
	</ul>
</div>

        <section class="social">
	
	<a href="https://twitter.com/ahmedrhasan" rel="me"><i class="fab fa-twitter fa-lg" aria-hidden="true"></i></a>
	
	
	
	<a href="https://github.com/aays" rel="me"><i class="fab fa-github fa-lg" aria-hidden="true"></i></a>
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	<a href="mailto:ahmed.hasan@mail.utoronto.com" rel="me"><i class="fas fa-at fa-lg" aria-hidden="true"></i></a>
	
</section>

      </div>
    </div>
    
<div class="copyright">
  &copy; 2021 htr3n
  
    <a href="https://creativecommons.org/licenses/by-sa/4.0">CC BY-SA 4.0</a>
  
</div>


<div class="builtwith">
Built with <a href="https://gohugo.io">Hugo</a> and <a href="https://github.com/htr3n/hyde-hyde">hyde-hyde</a>.
</div>


  </div>
</div>

        <div class="content container">
            
    
<article>
  <header>
    <h1>Recombination rate variation and linked selection in C. reinhardtii v2.0 - a summary</h1>
    
    
<div class="post__meta">
    
    
      <i class="fas fa-calendar-alt"></i> Nov 5, 2018
    
    
    
      
      
          in
          
          
              <a class="badge badge-category" href="/categories/publications">PUBLICATIONS</a>
              
          
      
    
    
    
      
      
          <br/>
           <i class="fas fa-tags"></i>
          
          <a class="badge badge-tag" href="/tags/research">research</a>
           
      
          <a class="badge badge-tag" href="/tags/writing">writing</a>
           
      
          <a class="badge badge-tag" href="/tags/r">r</a>
          
      
    
    
    <br/>
    
</div>


  </header>
  
  
  
    <div class="toc-wrapper">
      <input type="checkbox" id="tocToggle">
      <label for="tocToggle">Table of Contents</label>
      
        <div class="toc" id="TableOfContents"></div>
      
    </div>
  
  
  <div class="post">
    
      <script type="text/javascript" async
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    displayMath: [['$$','$$']],
    processEscapes: true,
    processEnvironments: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre'],
    TeX: { equationNumbers: { autoNumber: "AMS" },
         extensions: ["AMSmath.js", "AMSsymbols.js"] }
  }
  });
  MathJax.Hub.Queue(function() {
    
    
    
    var all = MathJax.Hub.getAllJax(), i;
    for(i = 0; i < all.length; i += 1) {
        all[i].SourceElement().parentNode.className += ' has-jax';
    }
  });

  MathJax.Hub.Config({
  
  TeX: { equationNumbers: { autoNumber: "AMS" } }
  });
</script>

    
    <p>In June of this year, I uploaded my first first-author (there has to be a cleaner way of saying that) <a href="https://www.biorxiv.org/content/early/2018/11/05/340992">manuscript</a> to the preprint server <a href="biorxiv.org">BioRxiv</a>. This was a huge and important milestone for me, and I was incredibly excited to have it out for the world to see virtually as soon as it was done &ndash; such is the power of preprint servers!</p>
<p>Like in most endeavours in science, however, there&rsquo;s always room for improvement, and I&rsquo;ve been very fortunate to have gotten some thorough feedback from both members of my committee as well as other researchers in the community at large. Here, I&rsquo;m going to briefly summarise the paper before covering what&rsquo;s been changed so far in this iteration of the manuscript and why.</p>
<h2 id="overview">Overview</h2>
<p>In this paper, we looked at the landscape of recombination rate variation across the genome of the haploid unicellular alga <em>Chlamydomonas reinhardtii</em>, asking the questions: 1) What genomic features predict recombination rate variation? 2) How does recombination rate correlate with nucleotide diversity? 3) What is the rate of sex in natural populations of <em>C. reinhardtii</em>?</p>
<p>To obtain genome-wide estimates of recombination rate, we used a dataset of whole-genome sequences from 19 <em>C. reinhardtii</em> individuals sampled in Quebec. Next, we used the recombination rate estimator <a href="https://journals.plos.org/plosgenetics/article?id=10.1371/journal.pgen.1003090">LDhelmet</a>, which calculates historical recombination rate using statistical associations between alleles.</p>
<h2 id="initial-results">(Initial) Results</h2>
<h3 id="recombination-rate-is-highest-immediately-surrounding-genes">Recombination rate is highest immediately surrounding genes</h3>
<p>In examining which genome annotations had the highest recombination rates, we subdivided intergenic sequence into regions 2 kb upstream or downstream of a gene, while subdividing genomic sequence into UTR regions, introns, and protein-coding sequence. Here, recombination was indeed highest in the 2 kb of sequence immediately flanking genes, in accordance with observations in other non-mammalian species. (Fig. 2 in the paper)</p>
<h3 id="recombination-rate-is-correlated-with-nucleotide-diversity">Recombination rate is correlated with nucleotide diversity</h3>
<p>In the absence of recombination, selection at linked sites can result in the loss of linked neutral variation surrounding a selected site. In the initial iteration of this manuscript, we reported a positive association between recombination rate and nucleotide diversity. However, this analysis bears the risk of potential autocorrelation; I explain further below. (Fig. 3 in the paper)</p>
<h3 id="recombination-is-correlated-with-gc-content">Recombination is correlated with GC content</h3>
<p>Despite mutation being AT-biased in <em>C. reinhardtii</em> (Ness 2015, Genome Res.) the genome is incredibly GC-rich (~64%). One possible explanation for this high GC content is the action of GC-biased gene conversion (gBGC), wherein mismatch repair following a recombination event preferentially fixes GC alleles, effectively mimicking positive selection. We did find a weakly positive correlation between GC content and recombination in <em>C. reinhardtii</em> &ndash; however, since LDhelmet does not discern between crossover events and gene conversions, more precise inferences regarding gBGC will require further work.</p>
<h3 id="the-frequency-of-sex-in-_c-reinhardtii_">The frequency of sex in <em>C. reinhardtii</em></h3>
<p><em>C. reinhardtii</em> is a facultatively sexual organism, in that it primarily reproduces asexually and only switches to a sexual mode of reproduction when nutrient stressed. Thus, <em>effective</em> recombination is modulated by the frequency of meiosis relative to mitosis. By combining our genome-wide estimate of recombination rate alongside prior estimates of neutral diversity, mutation rate, and physical recombination rate, we obtained a ratio of meiosis occuring every ~770 mitotic generations.</p>
<h2 id="manuscript-20">Manuscript 2.0</h2>
<p>In this iteration of the manuscript, the following things have been changed/added:</p>
<h3 id="recombination-rate-and-diversity---autocorrelation">Recombination rate and diversity - autocorrelation?</h3>
<p>LDhelmet estimates <em>population</em> recombination rate <em>ρ</em>, which is equivalent to <em>2N<!-- raw HTML omitted -->e<!-- raw HTML omitted -->r</em>, where <em>N<!-- raw HTML omitted -->e<!-- raw HTML omitted --></em> is effective population size (see Charlesworth 2009, Nature, for a useful review) while <em>r</em> represents <em>physical</em> recombination rate. Meanwhile, diversity (<em>θ</em>) is equivalent to <em>2N<!-- raw HTML omitted -->e<!-- raw HTML omitted -->μ</em>, where <em>μ</em> is the mutation rate.</p>
<p>What this means is that provided a sufficiently large effective population size (indeed the case in <em>C. reinhardtii</em> - see Ness 2015), it could in fact be <em>variance in N<!-- raw HTML omitted -->e<!-- raw HTML omitted --> across the genome</em> that is driving the observed correlation, and not the effects of selection at linked sites. This was pointed out to me by both my committee members as well as <a href="http://www.twitter.com/tylervkent">Tyler Kent</a>, and effectively meant that one of the central premises of the paper was flawed.</p>
<p>To circumvent this, I initially attempted to correlate diversity with estimates of physical recombination rate from the linkage map of <em>C. reinhardtii</em>, and found no significant correlation. This was a bit of a blow; the original correlation was arguably the main finding of the paper in my eyes, and the basis for claiming that linked selection was affecting levels of diversity in the genome &ndash; an effect pronounced further still by relatively infrequent sex.</p>
<p>My first instinct was to look to variance in functional density, which has previously explained the lack of a correlation between physical recombination rate and diversity in a number of plant species (Wright et al 2006, Genetics; Flowers et al 2012, Mol Biol Evol). An inverse correlation between diversity and functional density indicates that regions with more &lsquo;targets for selection&rsquo; display reduced variation, thus still suggesting the action of linked selection. Instead, however, I found a <em>positive</em> correlation between the two. This has previously been observed in chicken (Mugal 2015), but explained as a statistical artifact. Ultimately, the issue remained unresolved. The only other possibility was that the linkage map data were too sparse to make inferences surrounding linked selection with &ndash; but that still remained an unsatisfying answer.</p>
<p>(Bonus: This saga has since been immortalized in my <a href="https://github.com/aays/evol2018-poster">Evolution 2018 poster</a>, presented in Montpellier, France &ndash; have a gander!)</p>
<p>While I was busy struggling to rewrite the manuscript to reflect these new developments, my PI obtained crossover data from a recent paper that had directly identified recombination events through whole-genome sequencing of recombinants (Liu et al 2017, Nature Ecol Evol). Using crossover density as a proxy for physical recombination rate, Rob found that it correlated with <em>both</em> the LDhelmet estimates of recombination rate as well as nucleotide diversity. What this meant was the LDhelmet recombination rate estimates were indeed reflective of variation in recombination rate and not just <em>N<!-- raw HTML omitted -->e<!-- raw HTML omitted --></em> and that the effect of linked selection on diversity was indeed real.</p>
<p>The updated version of the manuscript now contains both explanations of the above as well as these newer results. I found this to be a very important lesson in the value of getting good feedback, even if it isn&rsquo;t necessarily what one might want to hear in the moment.</p>
<h3 id="a-technical-note---some-cosmetic-changes-to-figures">A technical note - some cosmetic changes to figures</h3>
<p>Anyone who knows me knows that I&rsquo;m a bit of a data visualization buff, and spend far too much time finding ways to make plots look both prettier and more intuitive in their presentation. With this in mind, I was not particularly satisfied with the initial iteration of Figure 1, which displayed a) the distribution of recombination rates across the 17 chromosomes of <em>C. reinhardtii</em> and b) the decay of linkage disequilibrium across the genome, once again split by chromosome. Both plots were shaded by chromosome length, which was on a purple to blue scale &ndash; arguably not the most intuitive! Furthermore, the font sizes of the axis titles and labels were inconsistent, with some of the labels being harder to read.</p>
<p>The new version of the plot features more intuitive colours (yellow to orange/red, via the <code>wesanderson</code> package in R) and larger axis tick labels, which I think make for a more easily understandable plot. However, the main reason I&rsquo;m excited about this is that this was my first use of the <code>patchwork</code> package (<a href="https://github.com/thomasp85/patchwork">obtainable here</a>) for stitching together plots in R, which works beautifully with <code>ggplot2</code> 3.0&rsquo;s new <code>labs(tag)</code> feature &ndash; both of which I haven&rsquo;t seen nearly enough fanfare about for the utility they provide.</p>
<p>Namely, where in the past stitching together plots has required either some <code>gridExtra</code> wizardry or simply exporting to Photoshop/Illustrator/what have you, these packages allow for the entire process to be done completely in R. To create a new plot featuring two plot objects <code>p1</code> and <code>p2</code> side by side, <code>patchwork</code> allows us to simply write</p>
<div class="highlight"><pre style="color:#f8f8f2;background-color:#272822;-moz-tab-size:4;-o-tab-size:4;tab-size:4"><code class="language-R" data-lang="R">p1 <span style="color:#f92672">+</span> p2
</code></pre></div><p>and we&rsquo;re good to go. It seems so intuitive and in line with the <code>ggplot2</code> philosophy I&rsquo;m surprised it hasn&rsquo;t received even more attention.</p>
<p>The second component of this is that the newest version of <code>ggplot2</code> features a new <code>tag</code> argument in the <code>labs</code> function (i.e. <code>labs(tag = 'A')</code>) which allows for the easy addition of tags to the top left corner of plots. Together, these constitute a very powerful toolkit for making plots of this sort all within R itself.</p>
<p>Finally, Figure 3 has also been updated to reflect the analyses described above regarding crossover density from the Liu dataset. Code for all figures in the paper can be found <a href="https://github.com/aays/2018-ld-paper/blob/master/plots.R">here</a>.</p>
<h3 id="estimating-the-frequency-of-sex-with-population-genetic-methods---a-better-explanation">Estimating the frequency of sex with population genetic methods - a better explanation</h3>
<p>Previously, our calculation of the rate of sex was explained in terms of &lsquo;effective mitoses&rsquo; and &lsquo;effective meioses&rsquo;, with the effects of mutation and recombination manifesting as either, respectively. This explanation was somewhat confusing, and so it has been rephrased entirely in terms of the formulae underlying the calculation, where the physical recombination rate <em>r</em> is taken to be <em>rf</em>, where <em>f</em> is the frequency of sex in this system. While the result remains unchanged, we hope this makes the calculation clearer for readers.</p>
<h2 id="to-sum-up">To sum up</h2>
<p>This manuscript has been in prep since about three months into my graduate school career, and has undergone various changes and makeovers as further analyses were added. Beyond that, there still is room for improvement, and I&rsquo;ll be looking for ways to update the manuscript further. As someone with little to no population genetics background upon starting graduate school, it&rsquo;s certainly been challenging putting this work together thus far, but equally as rewarding at the same time.</p>
<p>Of course, I&rsquo;d love to hear any further thoughts and comments on the manuscript if you have any &ndash; please feel more than welcome to reach out via email or on Twitter!</p>

    <script>
    
    
      var tags = document.getElementsByTagName("h2");
      for (var i=0, h2; h2 = tags[i]; i++) {
        h2.id = h2.innerHTML.toLowerCase().replace(" ", "-");
      }
      var tags = document.getElementsByTagName("h3");
      for (var i=0, h3; h3 = tags[i]; i++) {
        h3.id = h3.innerHTML.toLowerCase().replace(" ", "-");
      }
      var tags = document.getElementsByTagName("h4");
      for (var i=0, h4; h4 = tags[i]; i++) {
        h4.id = h4.innerHTML.toLowerCase().replace(" ", "-");
      }
    </script>
  </div>
  


  

  
    


</article>


        </div>
        
    

<script defer src="https://use.fontawesome.com/releases/v5.5.0/js/all.js" integrity="sha384-GqVMZRt5Gn7tB9D9q7ONtcp4gtHIUEW/yG7h98J7IpE3kpi+srfFyyB/04OV6pG0" crossorigin="anonymous"></script>


    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.13.1/highlight.min.js"></script>
        
            
                <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.13.1/languages/r.min.js"></script>
            
                <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.13.1/languages/python.min.js"></script>
            
                <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.13.1/languages/bash.min.js"></script>
            
        
    <script type="text/javascript">
        
        hljs.configure({languages: ["r, python, bash"]});
        
        hljs.initHighlightingOnLoad();
    </script>
    



<script src="https://cdnjs.cloudflare.com/ajax/libs/tocbot/4.4.2/tocbot.js"></script>
<script type="text/javascript">
  if (tocbot) {
    tocbot.init({
      
      tocSelector: '.toc',
      
      contentSelector: '.post',
      
      headingSelector: 'h2, h3, h4',
      collapseDepth: 4
    });
  }
</script>



    



    </body>
</html>
