<!DOCTYPE html>
<html lang="en">
    
    


    <head>
    <link href="https://gmpg.org/xfn/11" rel="profile">
    <meta charset="utf-8">
<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="Cache-Control" content="public" />
<!-- Enable responsiveness on mobile devices -->
<meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1">
<meta name="generator" content="Hugo 0.74.3" />

    
    
    

<title>CV • Ahmed Hasan</title>


<meta name="twitter:card" content="summary"/>
<meta name="twitter:title" content="CV"/>
<meta name="twitter:description" content="Education   Ph.D. - University of Toronto Mississauga - 2017-present
 Computational &amp; Evolutionary Genomics - Dept. of Cell and Systems Biology Supervisor: Prof. Rob Ness    B.Sc. (Hons.) - University of Toronto - 2012-2016 Majors in Human Biology, Cell &amp; Systems Biology Thesis: Study of the endophytic bacterial and fungal communities inhabiting coconut palms infected by lethal yellowing-like phytoplasmas in Côte d’Ivoire. Supervisors: Prof. Keiko Yoshioka &amp; Dr."/>

<meta property="og:title" content="CV" />
<meta property="og:description" content="Education   Ph.D. - University of Toronto Mississauga - 2017-present
 Computational &amp; Evolutionary Genomics - Dept. of Cell and Systems Biology Supervisor: Prof. Rob Ness    B.Sc. (Hons.) - University of Toronto - 2012-2016 Majors in Human Biology, Cell &amp; Systems Biology Thesis: Study of the endophytic bacterial and fungal communities inhabiting coconut palms infected by lethal yellowing-like phytoplasmas in Côte d’Ivoire. Supervisors: Prof. Keiko Yoshioka &amp; Dr." />
<meta property="og:type" content="article" />
<meta property="og:url" content="https://aays.github.io/vitae/" />
<meta property="og:site_name" content="Title" />


    


<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/github.min.css">








<link rel="stylesheet" href="/scss/hyde-hyde.4e02f3d31db978c29e43cbdee3af32ed98d01bfbdf98034eee6f1408857c85bf.css" integrity="sha256-TgLz0x25eMKeQ8ve468y7ZjQG/vfmANO7m8UCIV8hb8=">


<link rel="stylesheet" href="/scss/print.2744dcbf8a0b2e74f8a50e4b34e5f441be7cf93cc7de27029121c6a09f9e77bc.css" integrity="sha256-J0Tcv4oLLnT4pQ5LNOX0Qb58&#43;TzH3icCkSHGoJ&#43;ed7w=" media="print">




<link rel="stylesheet" href="/scss/tocbot.5ef07cebc3c477b54270456f149ee02922479bb7555fd344b2c69f953b0e7e5e.css" integrity="sha256-XvB868PEd7VCcEVvFJ7gKSJHm7dVX9NEssaflTsOfl4=">



    <!-- HTML5 shim and Respond.js for IE8 support of HTML5 elements and media queries -->
    <!--[if lt IE 9]>
    <script src="https://oss.maxcdn.com/html5shiv/3.7.3/html5shiv.min.js"></script>
    <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
    <![endif]-->
    <!-- Icons -->
    <link rel="apple-touch-icon-precomposed" sizes="144x144" href="/apple-touch-icon-144-precomposed.png">
    <link rel="shortcut icon" href="/favicon.png">
    
    

</head>


    <body class=" ">
    
<div class="sidebar">
  <div class="container ">
    <div class="sidebar-about">
      <span class="site__title">
        <a href="https://aays.github.io/">Ahmed Hasan</a>
      </span>
      
        
        
        
        <div class="author-image">
          <img src="https://aays.github.io/img/profile.jpeg" alt="Author Image" class="img--circle img--headshot element--center">
        </div>
        
      
      
      <p class="site__description">
        
      </p>
    </div>
    <div class="collapsible-menu">
      <input type="checkbox" id="menuToggle">
      <label for="menuToggle">Ahmed Hasan</label>
      <div class="menu-content">
        <div>
	<ul class="sidebar-nav">
		 
		 
			 
				<li>
					<a href="/about/">
						<span>About</span>
					</a>
				</li>
			 
		 
			 
				<li>
					<a href="/post/">
						<span>Posts</span>
					</a>
				</li>
			 
		 
			 
				<li>
					<a href="/projects/">
						<span>Projects</span>
					</a>
				</li>
			 
		 
			 
				<li>
					<a href="/vitae/">
						<span>CV</span>
					</a>
				</li>
			 
		
	</ul>
</div>

        <section class="social">
	
	<a href="https://twitter.com/ahmedrhasan" rel="me"><i class="fab fa-twitter fa-lg" aria-hidden="true"></i></a>
	
	
	
	<a href="https://github.com/aays" rel="me"><i class="fab fa-github fa-lg" aria-hidden="true"></i></a>
	
	
	
	
	
	
	
	
	
	
	
	
	
	
	<a href="mailto:ahmed.hasan@mail.utoronto.com" rel="me"><i class="fas fa-at fa-lg" aria-hidden="true"></i></a>
	
</section>

      </div>
    </div>
    
<div class="copyright">
  &copy; 2020 htr3n
  
    <a href="https://creativecommons.org/licenses/by-sa/4.0">CC BY-SA 4.0</a>
  
</div>


<div class="builtwith">
Built with <a href="https://gohugo.io">Hugo</a> and <a href="https://github.com/htr3n/hyde-hyde">hyde-hyde</a>.
</div>


  </div>
</div>

        <div class="content container">
            
    
<article>
  <header>
    <h1>CV</h1>
    
    
<div class="post__meta">
    
    
    
    
    
    
    <br/>
    
</div>


  </header>
  
  
  
    <div class="toc-wrapper">
      <input type="checkbox" id="tocToggle">
      <label for="tocToggle">Table of Contents</label>
      
        <div class="toc" id="TableOfContents"></div>
      
    </div>
  
  
  <div class="post">
    
      <script type="text/javascript" async
  src="https://cdn.mathjax.org/mathjax/latest/MathJax.js?config=TeX-AMS-MML_HTMLorMML">
  MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [['$','$'], ['\\(','\\)']],
    displayMath: [['$$','$$']],
    processEscapes: true,
    processEnvironments: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre'],
    TeX: { equationNumbers: { autoNumber: "AMS" },
         extensions: ["AMSmath.js", "AMSsymbols.js"] }
  }
  });
  MathJax.Hub.Queue(function() {
    
    
    
    var all = MathJax.Hub.getAllJax(), i;
    for(i = 0; i < all.length; i += 1) {
        all[i].SourceElement().parentNode.className += ' has-jax';
    }
  });

  MathJax.Hub.Config({
  
  TeX: { equationNumbers: { autoNumber: "AMS" } }
  });
</script>

    
    <h2 id="education">Education</h2>
<ul>
<li>
<p><strong>Ph.D. - University of Toronto Mississauga</strong> - <em>2017-present</em></p>
<ul>
<li>Computational &amp; Evolutionary Genomics - Dept. of Cell and Systems Biology</li>
<li><em>Supervisor:</em> Prof. Rob Ness</li>
</ul>
</li>
<li>
<p><strong>B.Sc. (Hons.) - University of Toronto</strong>  - <em>2012-2016</em><!-- raw HTML omitted --></p>
<ul>
<li>Majors in Human Biology, Cell &amp; Systems Biology<!-- raw HTML omitted --></li>
<li><em>Thesis:</em> Study of the endophytic bacterial and fungal communities inhabiting coconut palms infected by lethal yellowing-like phytoplasmas in Côte d’Ivoire.<!-- raw HTML omitted --></li>
<li><em>Supervisors:</em> Prof. Keiko Yoshioka &amp; Dr. Yaima Arocha Rosete</li>
</ul>
</li>
</ul>
<h2 id="publications">Publications</h2>
<h3 id="graduate-research">Graduate Research</h3>
<p>Craig RJ, <strong>Hasan AR</strong>, Ness RW, Keightley PD (2020). Comparative genomics of <em>Chlamydomonas</em>. <em>bioRxiv</em>. <a href="https://www.biorxiv.org/content/10.1101/2020.06.13.149070v1">Available online.</a> (Submitted)</p>
<p><strong>Hasan AR</strong>, Ness RW (2020). Recombination rate variation and infrequent sex influence genetic diversity in <em>Chlamydomonas reinhardtii</em>. <em>Genome Biology and Evolution</em>. 12(4), 370-380. doi:10.1093/gbe/evaa057. <a href="https://academic.oup.com/gbe/advance-article/doi/10.1093/gbe/evaa057/5809146?searchresult=1">Available online.</a> (<a href="https://www.biorxiv.org/content/early/2018/06/07/340992">Preprint</a>)</p>
<p><strong>Hasan AR</strong>, Duggal JK, Ness RW. (2019) Consequences of recombination for the evolution of the mating type locus in <em>Chlamydomonas reinhardtii</em>. <em>New Phytologist</em>. 224(3), 1339-1348. doi:10.1111/nph.16003. <a href="https://nph.onlinelibrary.wiley.com/doi/10.1111/nph.16003">Available online.</a> (<a href="https://www.biorxiv.org/content/10.1101/565275v1.abstract">Preprint</a>)</p>
<h3 id="pedagogical-works">Pedagogical Works</h3>
<p>Johnston LW, Bonsma-Fisher M, Ostblom J, <strong>Hasan AR</strong>, Santangelo JS, Coome L, Tran L, Sales de Andrade E, Mahallati S. A graduate student-led participatory live-coding quantitative methods course in R: Experiences on initiating, developing, and teaching. <em>Journal of Open Source Education</em>. 2(16), 49. doi:10.21105/jose.00049. <a href="https://jose.theoj.org/papers/10.21105/jose.00049">Available online.</a></p>
<p>Bonsma-Fisher M, <strong>Hasan AR</strong> (2018). Working with plant phenology data and fitting a nonlinear model using least squares in R. NEON Faculty Mentoring Network, <em>QUBES Educational Resources</em>. doi:10.25334/Q4Q73D. <a href="https://qubeshub.org/qubesresources/publications/978/1">Available online.</a></p>
<h3 id="undergraduate-research">Undergraduate Research</h3>
<p>Morales-Lizcano NP, <strong>Hasan AR</strong>, To H, Lekadou TT, Copeland J, Wang PW, Diallo HA, Konan Konan JL, Yoshioka K, Moeder W, Scott J, Arocha Rosete Y (2017). Microbial diversity in leaves, trunk and rhizosphere of coconut palms (<em>Cocos nucifera</em> L.) associated with the coconut lethal yellowing phytoplasma in Grand-Lahou, Côte d’Ivoire. <em>African Journal of Biotechnology</em>. 16(27), 1534-1550. doi:10.5897/AJB2017.16006. <a href="http://www.academicjournals.org/journal/AJB/article-abstract/BE373F465038">Available online.</a></p>
<p>Arocha Rosete Y, Morales-Lizcano NP, <strong>Hasan AR</strong>, Yoshioka K, Moeder W, Michelutti R, Satta E, Bertaccini A, Scott J (2016). First report of the identification of a ‘Candidatus Phytoplasma pruni’-related strain in Trillium species in Canada. <em>New Disease Reports</em>, 34, 19. doi:10.5197/j.2044-0588.2016.034.019. <a href="https://www.ndrs.org.uk/article.php?id=034019#">Available online.</a></p>
<h3 id="oral-presentations">Oral Presentations</h3>
<ul>
<li><strong>Hasan AR</strong>, Craig RJ, Duggal JK, Keightley PD, Ness RW. Consequences of recombination for the evolution of the mating type locus in <em>Chlamydomonas reinhardtii</em>. Botany 2020 Virtual Conference; 2020***</li>
<li><strong>Hasan AR</strong>, Duggal JK, Ness RW. Consequences of recombination for the evolution of the mating type locus in <em>Chlamydomonas reinhardtii</em>. Evolution 2019 @ Providence, RI, United States; 2019</li>
<li><strong>Hasan AR</strong>, Ness RW. Genomic correlates of recombination rate variation in <em>Chlamydomonas reinhardtii</em>. CSB Research Day, Dept of Cell and Systems Biology @ University of Toronto, Toronto, ON; 2018. <strong>Awarded Best Oral Presentation</strong>.</li>
<li><strong>Hasan AR</strong>, Ness RW. Recombination rate variation in <em>Chlamydomonas reinhardtii</em>. Graduate Research Colloquium @ University of Toronto Mississauga, Mississauga, ON; 2018</li>
<li><strong>Hasan AR</strong>, Ness RW. The landscape of recombination rate variation in <em>Chlamydomonas reinhardtii</em>. Dept of Cell and Systems Biology Seminar Series @ University of Toronto, Toronto, ON; 2017</li>
<li><strong>Hasan AR</strong>, Ness RW. Recombination rate variation and leaky organelle inheritance in <em>Chlamydomonas reinhardtii</em>. <a href="https://midwestpopgen.wordpress.com/2017/05/02/midwest-popgen-2017-michigan-state-university/">Midwest PopGen 2017</a> @ Michigan State University, East Lansing, MI; 2017</li>
<li>Chan SSW, <strong>Hasan AR</strong>. Research: more than just a plan B. <a href="http://operationmedschool.com/toronto/">Operation Med School Toronto</a>/STEM Fellowship; 2017</li>
<li>Noukhovitch A, <strong>Hasan AR</strong>. A new generation of data-native learners. <a href="civictech.ca">Civic Tech Toronto</a>; 2016. (<a href="https://www.youtube.com/watch?v=yhOrvPaNqRY">Video</a>)</li>
</ul>
<p>*** <em>Conference originally to be held in Anchorage, AK, but held virtually due to COVID-19 outbreak</em></p>
<h3 id="poster-presentations">Poster Presentations</h3>
<ul>
<li>
<p><strong>Hasan AR</strong>, Ness RW. Recombination rate variation and infrequent sex influence genetic diversity in <em>Chlamydomonas reinhardtii</em>. SMBE 2020 @ Quebec City, QC, Canada; 2020***</p>
</li>
<li>
<p><strong>Hasan AR</strong>, Duggal JK, Ness RW. Consequences of recombination for the evolution of the mating type locus in <em>Chlamydomonas reinhardtii</em>. SMBE 2020 @ Quebec City, QC, Canada; 2020***</p>
</li>
<li>
<p><strong>Hasan AR</strong>, Ness RW. Recombination rate variation and linked selection in the <em>Chlamydomonas reinhardtii</em> genome. Joint Congress on Evolutionary Biology II @ Montpellier, France; 2018.</p>
</li>
</ul>
<p>*** <em>Abstract accepted, but conference cancelled due to COVID-19 outbreak</em></p>
<h3 id="software-packages">Software Packages</h3>
<p>Liu J, <strong>Hasan AR</strong>, Ness RW. readcomb - fast detection of recombinant reads in BAM files. Currently in development: <a href="https://github.com/ness-lab/readcomb">GitHub repo</a></p>
<p>Johnston LW, Ostblom J, <strong>Hasan AR</strong>. ROStools: Reproducible and open science toolkit. Currently in development: <a href="https://gitlab.com/rostools">GitLab repo</a></p>
<h2 id="teaching-mentoring-and-course-development">Teaching, Mentoring, and Course Development</h2>
<h3 id="course-instructorships">Course instructorships</h3>
<ul>
<li><em><strong>Course Instructor</strong></em>, Quantitative Methods in R for Biology (<a href="https://uoftcoders.github.io/rcourse/">EEB313</a>), Department of Ecology and Evolutionary Biology, University of Toronto; 2018-2019
<ul>
<li>Responsible for teaching first quarter of course, grading relevant assignments, and assisting students with projects</li>
<li>Developed and taught module on Reproducible science with Git and GitHub</li>
<li>(2018) Responsible for course organization and course administration</li>
<li>Topics taught - introductory R, <code>dplyr</code>, <code>tidyr</code>, <code>ggplot2</code>, Git/GitHub</li>
</ul>
</li>
</ul>
<h3 id="teaching-assistantships">Teaching assistantships</h3>
<ul>
<li><em><strong>Teaching Assistant</strong></em>, Bioinformatics (<a href="https://student.utm.utoronto.ca/calendar/course_detail.pl?Depart=3&amp;Course=BIO362H5">BIO362H5F</a>), Department of Biology, University of Toronto Mississauga; Fall 2017, Spring 2019, Fall 2019 (marking only)
<ul>
<li>Assisted undergraduate students in learning Python for bioinformatics; graded labs; designed a lab introducing the <code>pandas</code> package.</li>
<li>Other packages taught: <code>Bio.SeqIO</code>, <code>Bio.Phylo</code>, <code>re</code>, <code>pysam</code>, <code>numpy</code>.</li>
</ul>
</li>
<li><em><strong>Teaching Assistant</strong></em>, Genomics (<a href="https://student.utm.utoronto.ca/calendar/course_detail.pl?Depart=3&amp;Course=BIO458H5">BIO458H5S</a>), <a href="utm.utoronto.ca/biology">Department of Biology</a>, University of Toronto Mississauga; 2018
<ul>
<li>Assisted graduate and undergraduate learners in navigating a Linux/Python environment for genomics analyses</li>
<li>Software taught: <code>samtools</code>, <code>sickle</code>, GATK, <code>tophat</code>, <code>cufflinks</code>, <code>SPAdes</code></li>
</ul>
</li>
</ul>
<h3 id="course-development">Course development</h3>
<ul>
<li><em><strong>Quantitative Curriculum/Course Developer</strong></em>, Department of Biology, University of Toronto Mississauga; 2020
<ul>
<li>Designing introductory R material for first year labs at the University of Toronto Mississauga, as part of broader initiative to introduce quantitative and computational literacy skills into Biology&rsquo;s undergraduate program</li>
</ul>
</li>
<li><em><strong>Course Developer</strong></em>, Genomics (<a href="https://student.utm.utoronto.ca/calendar/course_detail.pl?Depart=3&amp;Course=BIO458H5">BIO458H5S</a>), Department of Biology, University of Toronto Mississauga; 2017
<ul>
<li>Designed course labs in JupyterHub environment alongside course instructor; topics included reference-based assembly, variant calling, transcriptomics, and de novo assembly</li>
</ul>
</li>
</ul>
<h3 id="external-teaching">External teaching</h3>
<ul>
<li><em><strong>President</strong></em>, <a href="http://uoftcoders.github.io">University of Toronto Coders</a>; 2019-present.
<ul>
<li>Overseeing the U of T Coders council and all organization activities.</li>
<li>Helped initiate a peer mentorship program (2020)</li>
</ul>
</li>
<li><em><strong>Co-instructor</strong></em>, Introduction to R and Graphics for Biology, <a href="https://www.physiology.utoronto.ca/international-exchange-developmental-biology">International Exchange Program in Developmental and Perinatal Biology</a>, University of Toronto Dept. of Physiology + Karolinska Institute; 2018
<ul>
<li>Topics - Introduction to R, plotting, compiling R markdown files, <code>dplyr</code></li>
</ul>
</li>
<li><em><strong>Instructor/Contributor</strong></em>, Programming workshops + code-alongs, <a href="http://uoftcoders.github.io">University of Toronto Coders</a>; 2017-present.
<ul>
<li>Taught and designed various coding workshops (R, Python)</li>
</ul>
</li>
<li><em><strong>Helper</strong></em>, <a href="www.software-carpentry.org">Software Carpentry</a> workshops (Python). <a href="https://uoftcoders.github.io/2017-02-03-utoronto/">Event website</a>; 2017</li>
</ul>
<h4 id="workshops-taught-with-the-carpentries">Workshops taught with the Carpentries</h4>
<ul>
<li><em><strong>Instructor</strong></em>, <a href="www.software-carpentry.org">Software Carpentry</a> workshops (Python), University of Toronto. <a href="https://uoftcarpentries.github.io/2020-02-20-utsg/">Event Website</a>; 2020
<ul>
<li>Workshop taught in affiliation with U of T Libraries</li>
<li>Topics - The Unix Shell</li>
</ul>
</li>
<li><em><strong>Instructor</strong></em>, <a href="www.datacarpentry.com">Data Carpentry</a> workshops (R), University of Toronto. <a href="https://mikeserafin.github.io/2020-02-10-utm/">Event Website</a>; 2020
<ul>
<li>Workshop taught in affiliation with U of T Libraries</li>
<li>Topics - data analysis and visualization with R using <code>dplyr</code> and <code>ggplot2</code></li>
</ul>
</li>
<li><em><strong>Instructor</strong></em>, <a href="www.datacarpentry.com">Data Carpentry</a> workshops (Python), University of Toronto. <a href="https://uoftcoders.github.io/2018-09-10-utoronto/">Event Website</a>; 2018
<ul>
<li>Topics - data analysis and visualization with <code>matplotlib</code> and <code>seaborn</code></li>
</ul>
</li>
<li><em><strong>Instructor</strong></em>, <a href="www.datacarpentry.org">Data Carpentry</a> workshops (Genomics), Centre for Addiction and Mental Health (CAMH) Toronto. <a href="https://canghel.github.io/2018-05-22-camh/">Event Website</a>; 2018
<ul>
<li>Taught the Data Wrangling in Genomics session in a two-day workshop for researchers both at CAMH and around the GTA.</li>
<li>Topics - quality control of sequencing reads, <code>FastQC</code>, read trimming with <code>trimmomatic</code>, alignment with <code>bwa</code>, variant calling</li>
</ul>
</li>
<li><em><strong>Instructor</strong></em>, <a href="www.software-carpentry.org">Software Carpentry</a> workshops (R), University of Toronto. <a href="https://uoftcoders.github.io/2018-01-18-utoronto/">Event website</a>; 2018
<ul>
<li>Taught the second R session in a two-day workshop for researchers and students at the University of Toronto.</li>
<li>Topics - loops, vectorization, <code>dplyr</code>, <code>ggplot2</code></li>
</ul>
</li>
</ul>
<h3 id="mentoring">Mentoring</h3>
<ul>
<li>Jimmy Liu and Gabriel El-Haddad (2020). Detecting signatures of recombination in pooled next-generation sequencing reads.</li>
<li>Eniolaye Balogun (2018-2019). Is recombination mutagenic? Scanning for mutations in next-generation sequence data.</li>
</ul>
<h2 id="awards">Awards</h2>
<ul>
<li><em>Rustom H. Dastur Graduate Scholarship</em>, Department of Cell and Systems Biology, University of Toronto; 2018
<ul>
<li>Value: $17,350. Award for graduate students in plant biology who successfully transfer into a PhD program, based on academic merit</li>
</ul>
</li>
<li><em>Conference Travel Grant</em>, School of Graduate Studies, University of Toronto; 2018
<ul>
<li>Value: $840. For attendance to the Joint Congress on Evolutionary Biology II, Montpellier, France</li>
</ul>
</li>
<li><em>Best Oral Presentation</em>, CSB Research Day 2018, Department of Cell and Systems Biology, University of Toronto; 2018
<ul>
<li>Value: $120.</li>
</ul>
</li>
<li><em>Elizabeth Ann Wintercorbyn Award</em>, Department of Cell and Systems Biology, University of Toronto; 2017
<ul>
<li>Value: $500. Research excellence award for a student in the department engaged in work with potential benefits for agriculture</li>
</ul>
</li>
</ul>
<h2 id="miscellaneous">Miscellaneous</h2>
<h3 id="programming-languagesenvironments">Programming languages/environments</h3>
<ul>
<li>Python</li>
<li>R (incl. <code>tidyverse</code> packages)</li>
<li>Bash/Unix shell</li>
<li>Jupyter (Notebook/Lab/Hub/<code>nbgrader</code>)</li>
<li>LaTeX</li>
<li>Git</li>
<li>JavaScript (familiarity)</li>
<li>ReactJS (familiarity)</li>
</ul>
<hr>
<p>More details regarding projects I&rsquo;ve worked on are available on the <a href="http://aays.github.io/projects">Projects</a> page.</p>

    <script>
    
    
      var tags = document.getElementsByTagName("h2");
      for (var i=0, h2; h2 = tags[i]; i++) {
        h2.id = h2.innerHTML.toLowerCase().replace(" ", "-");
      }
      var tags = document.getElementsByTagName("h3");
      for (var i=0, h3; h3 = tags[i]; i++) {
        h3.id = h3.innerHTML.toLowerCase().replace(" ", "-");
      }
      var tags = document.getElementsByTagName("h4");
      for (var i=0, h4; h4 = tags[i]; i++) {
        h4.id = h4.innerHTML.toLowerCase().replace(" ", "-");
      }
    </script>
  </div>
  


  

  
    


</article>


        </div>
        
    

<script defer src="https://use.fontawesome.com/releases/v5.5.0/js/all.js" integrity="sha384-GqVMZRt5Gn7tB9D9q7ONtcp4gtHIUEW/yG7h98J7IpE3kpi+srfFyyB/04OV6pG0" crossorigin="anonymous"></script>


    
    <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.13.1/highlight.min.js"></script>
        
            
                <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.13.1/languages/r.min.js"></script>
            
                <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.13.1/languages/python.min.js"></script>
            
                <script src="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.13.1/languages/bash.min.js"></script>
            
        
    <script type="text/javascript">
        
        hljs.configure({languages: ["r, python, bash"]});
        
        hljs.initHighlightingOnLoad();
    </script>
    



<script src="https://cdnjs.cloudflare.com/ajax/libs/tocbot/4.4.2/tocbot.js"></script>
<script type="text/javascript">
  if (tocbot) {
    tocbot.init({
      
      tocSelector: '.toc',
      
      contentSelector: '.post',
      
      headingSelector: 'h2, h3, h4',
      collapseDepth: 4
    });
  }
</script>



    



    </body>
</html>
